addons/medical_treatment/functions/fnc_createLitter.sqf

```diff
@@ -57,8 +57,15 @@ private _fnc_createLitter = {
             0
         ];
 
+        private _raycast = lineIntersectsSurfaces [_position vectorAdd [0,0,1], _position vectorAdd [0,0,-1e11], _patient, _medic, true, 1, "ROADWAY", "FIRE"];
+
+        _position = [_position, (_raycast # 0) # 0] select (_raycast isNotEqualTo []);
+        private _surfaceNormal = [[0, 0, 1], (_raycast # 0) # 1] select (_raycast isNotEqualTo []);
+
+        if (ASLToATL _position select 2 < -0.01) then {continue};
```

Comment:
no need for a continue here. invert the condition and move the event call into the if